R/bootstrap.R

```diff
@@ -948,8 +953,8 @@ textOutput <- function(outputId, container = if (inline) span else div, inline =
 #'   tableOutput("view")
 #' )
 #' @export
-verbatimTextOutput <- function(outputId) {
-  textOutput(outputId, container = pre)
+verbatimTextOutput <- function(outputId, placeholder = TRUE) {
+  textOutput(outputId, container = pre, class = if (!placeholder) "noplaceholder")
```

Comment:
I think it would make more sense to directly call `pre()` here instead of calling `textOutput()`.  This is so that you won't have to modify `textOuput` to accept `...`.